flocker/node/agents/blockdevice.py

```diff
@@ -408,6 +419,30 @@ def run(self, deployer):
         return succeed(None)
 
 
+def _ensure_no_filesystem(device):
+    """
+    Raises an error if there's already a filesystem on 'device'.
```

Comment:
rST for API docs (though we don't use or verify this in any way).  So probably double backtick instead of single quotes.